using_images/xpaas_images/fuse.adoc

```diff
@@ -105,6 +112,8 @@ $ oc login
 $ oc project <projectname>
 ----
 
+. Create a sample project using command `*mvn archetype:generate*`.
+
```

Comment:
Is this the complete command? It might need to be in the example box thing:

https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#formatting

So:

Create a sample project:

====
----
$ mvn archetype:generate
----
====